web-bundle/src/main/resources/com/graphhopper/maps/js/main-template.js

```diff
@@ -115,72 +119,23 @@ $(document).ready(function (e) {
         cmEditor.cm.execCommand('selectAll');
         cmEditor.cm.refresh();
     }
-    $("#custom-model-example").click(function() {
+
+    $("#custom-model-example").click(function () {
         showCustomModelExample();
         return false;
     });
 
-    var sendCustomData = function() {
-       mapLayer.clearElevation();
```

Comment:
clicking the custom model search button now triggers just a normal search, but before we send the request we check whether it should be custom or not.